### PR TITLE
TEXTSEARCH: Add @@ operator, remove useless overload of toTsVector

### PR DESCRIPTION
**Pull request by ranile**
Changes
- Make the code compile if `toTsVector` function is used
- Allow tsvector column to be compared with a tsquery

### CODE REVIEW

File: src/main/kotlin/epgx/functions/Comparisons.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package epgx.functions
+
+import epgx.ops.AtAtOp
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.CustomFunction
+
+/**
+ * Checks if [this] tsvector column is equals to some [tsQuery] query.
+ */
```
Comment (Benjozork) [inline src/main/kotlin/epgx/functions/Comparisons.kt]: It doesn't check if it's equals, it checks if it matches
```suggestion
/**
 * Checks if [this] tsvector column is matched by some [tsQuery] query.
 */
```

File: src/main/kotlin/epgx/ops/AtAtOp.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package epgx.ops
+
+import org.jetbrains.exposed.sql.ComparisonOp
+import org.jetbrains.exposed.sql.Expression
+
+/**
+ * Represents an SQL operator that checks if [vector] is equals to [query].
```
Comment (Benjozork) [inline src/main/kotlin/epgx/ops/AtAtOp.kt]:
```suggestion
 * Represents an SQL operator that checks if [query] matches [vector].
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: src/main/kotlin/epgx/functions/Comparisons.kt (hunk 1 above)
Comment (ranile) [reply src/main/kotlin/epgx/functions/Comparisons.kt]: Fixed

File: src/main/kotlin/epgx/ops/AtAtOp.kt (hunk 2 above)
Comment (ranile) [reply src/main/kotlin/epgx/ops/AtAtOp.kt]: It works both ways so i fixed the doc accordingly

**Comment by ranile** [comment]
Made the changes

File: src/main/kotlin/epgx/functions/Comparisons.kt [hunk 3]
```diff
@@ -5,7 +5,7 @@ import org.jetbrains.exposed.sql.Column
 import org.jetbrains.exposed.sql.CustomFunction
 
 /**
- * Checks if [this] tsvector column is equals to some [tsQuery] query.
+ * Checks if [this] tsvector column is matches to some [tsQuery] query.
```
Comment (Benjozork) [inline src/main/kotlin/epgx/functions/Comparisons.kt]:
```suggestion
 * Checks if [this] tsvector column matches [tsQuery] query.
```

File: src/main/kotlin/epgx/ops/AtAtOp.kt [hunk 4]
```diff
@@ -4,7 +4,7 @@ import org.jetbrains.exposed.sql.ComparisonOp
 import org.jetbrains.exposed.sql.Expression
 
 /**
- * Represents an SQL operator that checks if [vector] is equals to [query].
+ * Represents an SQL operator that checks if [vector] is matches to a [query] or vice versa.
```
Comment (Benjozork) [inline src/main/kotlin/epgx/ops/AtAtOp.kt]:
```suggestion
 * Represents an SQL operator that checks if [vector] matches [query].
```

**Review by Benjozork — CHANGES_REQUESTED** [review]

**Pull request merged**